lib/model/QubitInformationObject.php

```diff
@@ -1168,6 +1168,22 @@ public function getDigitalObject()
   }
 
   /**
+   * Get the digital object's checksum value
+   *
+   * @return string  digital object checksum or null
+   */
+  public function getDigitalObjectChecksum()
+  {
+    // Set digital object URL
+    $digitalObjects = $this->getDigitalObjects();
```

Comment:
I think there's a `QubitInformationObject::getDigitalObject()` that does basically what this code is doing. I'm actually not sure if getDigitalObject**s**() ever returns an array with more than one digital object.

https://github.com/artefactual/atom/blob/qa/2.3.x/lib/model/QubitInformationObject.php#L1142